aries-site/src/pages/components/layer/side-drawer-layer.mdx

```diff
@@ -0,0 +1,100 @@
+| **3** | **Right edge of window** | --                                         | --                                                                | The layer should be anchored to the right edge of the window and fill the height.                                                 |
+
+See [layer closing behavior](/components/layer#actionable) for details.
+
+<Notification
+  status="info"
+  message="A layer should either have actions in the footer or a close button in the header, not both."
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+### Grommet properties
+
+Apply these [layer properties](https://v2.grommet.io/layer) for a side drawer layer.
+
+| Property | Value    |
+| -------- | -------- |
+| position | right    |
+| full     | vertical |
+
+## Header behavior
+
+The default behavior of the layer header is to scroll with the layer content.
+
+However, actionable layers should have a sticky header that is either persistent or appears when the user scrolls up.
+
+### Persistent sticky header
+
+The Layer header and action buttons can be sticky while the rest of the Layer content is scrollable. This can be accomblished by wrapping a Form within a Box with `overflow` set to `auto`.
+
+<Example
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/LayerStickyExample.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/MonitorFormExample.js',
+  ]}
+  height={{ min: 'small' }}
+>
+  <LayerStickyExample />
+</Example>
+
+### Scroll up sticky header
+
+In order to maximize screen real-estate when content is scrollable, the header may scroll away as the user scrolls down but reappear as the user scrolls up. This can be accomblished by using the `sticky` prop
+on Heading and setting it to `"scrollup"`.
+
+<Example
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/LayerStickyScrollExample.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/MonitorFormExample.js',
+  ]}
+  height={{ min: 'small' }}
+>
+  <LayerStickyScrollExample />
+</Example>
+
+## Examples
+
+### Configuration form
+
+<Example
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/ConfigurationForm.js"
+  height={{ min: 'small' }}
+  template
+  showResponsiveControls={[]}
+>
+  <ConfigurationForm />
+</Example>
+
+### Filtering
+
+Click the filter icon to see the layer.
```

Comment:
Might be helpful to link back to [this section](https://deploy-preview-3028--keen-mayer-a86c8b.netlify.app/components/layer#overflow-behavior) on when the footer should be stickied or not

Reply:
Done.